src/V3Width.cpp

```diff
@@ -1377,6 +1377,55 @@ class WidthVisitor final : public VNVisitor {
         }
     }
 
+    void visit(AstSetuphold* nodep) override {
+        FileLine* const flp = nodep->fileline();
+
```

Comment:
Nit, please remove all the blank lines in this and in the next function.  Ok to keep one or two if really needed for intent - but don't seem necessary. Thanks.

I assume it's ok to merge once passes unless hear otherwise.

Reply:
Removed the blank lines